main.js

```diff
@@ -3,23 +3,28 @@ const navMenu = document.querySelector('.nav-menu');
 const close = document.querySelector('.close');
 const open = document.querySelector('.fa-bars');
 const navLink = document.querySelectorAll('.nav-link');
+const form = document.getElementById('form');
```

Comment:
- [ ] When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page. Currently both your `full name` and `email` stay auto-populated but the `textarea` doesn't so kindly fix this